install/db/achecker_schema.sql

```diff
@@ -308,7 +308,7 @@ CREATE TABLE `themes` (
   `title` varchar(80) NOT NULL DEFAULT '',
   `version` varchar(10) NOT NULL DEFAULT '',
   `dir_name` varchar(20) NOT NULL DEFAULT '',
-  `last_updated` date DEFAULT NOT NULL,
+  `last_updated` date NOT NULL,
```

Comment:
Let's add a `DEFAULT CURRENT_TIMESTAMP` for this not-null field. Can you do the same for other not-null date fields? Thanks.

Reply:
OK, would do that